bin/varnishtest/tests/m00017.vtc

```diff
@@ -20,6 +24,27 @@ server s1 {
 	expect req.url == "/qux"
 	expect req.http.foobar == "baz"
 	txresp
+
+	rxreq
+	txresp
+	rxreq
+	txresp
+	rxreq
+	txresp
+	rxreq
+	txresp
+	rxreq
+	txresp
+	rxreq
+	txresp
+	rxreq
+	txresp
+	rxreq
+	txresp
+	rxreq
+	txresp
+	rxreq
+	txresp
```

Comment:
I would rather redefine s1 or create a new server every time we reload VCL for a new use case.

We'd end up with a sequence like:

- set up sX
- load vclX
- run cX

Or close enough.